selery.yml

```diff
@@ -21,18 +23,22 @@ consider_releases: True
 releases_included: 3
 ## release weights
 release_weight: 70
+## weighted split over all contributors
+full_split: False
+## random weighted split with equal payout per contributor
+random_split: True
+## the amount of btc send per user at random_split
+btc_per_transaction: 0.000002
+## numbers of contributors payout per run at random split
+number_payout_contributors_per_run: 5
```

Comment:
This count includes dependency contributors right?

Reply:
Yes, maybe the comment should me more clear or the name better